### PR TITLE
Travis: fix build script for BGL examples

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes

This PR rectifies the build script of travis so BGL examples are tested despite their unusual architecture.

### CODE REVIEW

File: .travis/build_package.sh [hunk 1]
```diff
@@ -130,6 +130,14 @@ do
...
 	  cd $ROOT/$EXAMPLES
 	  build_examples
+  elif [ "$ARG" = BGL ]; then
+    EXAMPLES=$ROOT/BGL/examples
+    cd $EXAMPLES
+  	for dir in ./BGL*
+    do
+      cd $EXAMPLES/$dir
+      build_examples
+    done
```
Comment (lrineau) [inline .travis/build_package.sh]: That code should be the default for all packages.
There are several other similar cases:
```shellsession
$ ls BGL/examples Bounding_volumes/examples GraphicsView/demo Periodic_3_triangulation_3/demo Triangulation_3/demo

BGL/examples:
BGL_arrangement_2  BGL_LCC  BGL_OpenMesh  BGL_polyhedron_3  BGL_surface_mesh  BGL_triangulation_2

Bounding_volumes/examples:
Approximate_min_ellipsoid_d  Min_annulus_d  Min_circle_2  Min_ellipse_2  Min_quadrilateral_2  Min_sphere_d  Min_sphere_of_spheres_d  Rectangular_p_center_2

GraphicsView/demo:
Alpha_shapes_2      Bounding_volumes   Generator     icons                 Largest_empty_rect_2        Polygon    Segment_Delaunay_graph_2       Snap_rounding_2      Stream_lines_2
Apollonius_graph_2  Circular_kernel_2  GraphicsView  L1_Voronoi_diagram_2  Periodic_2_triangulation_2  resources  Segment_Delaunay_graph_Linf_2  Spatial_searching_2  Triangulation_2

Periodic_3_triangulation_3/demo:
Periodic_3_triangulation_3  Periodic_Lloyd_3

Triangulation_3/demo:
Triangulation_3  Triangulation_3_Geomview_demos
```

Comment (lrineau) [reply .travis/build_package.sh]: `for dir in ./BGL*`... You should test if `$dir` really is a directory, using `if [ -d $dir ]`.

**Comment by lrineau** [comment]
@maxGimeno, please rebase the patch on a recent version of `master`, so that we can see if it still work.

**Comment by maxGimeno** [comment]
done

**Comment by lrineau** [comment]
There is something fishy in the travis test, around here: https://travis-ci.org/CGAL/cgal/jobs/341835576#L6970

**Comment by lrineau** [comment]
Tested successfully in [CGAL-4.12-Ic-182], and then merged.

[CGAL-4.12-Ic-182]: https://cgal.geometryfactory.com/CGAL/testsuite/results-4.12-Ic-182.shtml

**Pull request merged**